site/cypress/e2e/cypress-feature/late_days_cache.spec.js

```diff
@@ -0,0 +1,429 @@
+                        cy.wrap(table).find('#Delete').first().click();
+                        cy.wait(2000);
+                        deleteLateDays();
+                    }
+                });
+            };
+            // Cleanup
+            deleteLateDays();
+            // View bulk late day changes
+            checkStudentsInCache();
+        });
+    });
+
+    describe('Test changes to late day extensions', () => {
+        it('should give extentions and check new status', () => {
+            cy.visit(['sample', 'extensions']);
+            cy.login('instructor');
+            cy.get('#gradeable-select').select('Late Allowed Homework');
+            for (const user_id of all_user_ids) {
+                // update the number of late days
+                cy.get('#user_id').type(user_id);
+                cy.get('#late-days').type(2, {force: true});
+                cy.get('#extensions-form')
+                    .find('a')
+                    .contains('Submit')
+                    .should('exist')
+                    .click();
+                cy.wait(2000);
+            }
+            checkStudentsInCache();
+            cy.logout();
+            CheckStatusUpdated(2,0);
+            cy.visit(['sample', 'extensions']);
+            cy.login('instructor');
+
+            const deleteExtensions = () => {
+                cy.get('body').then((body) => {
+                    if (body.find('#extensions-table').length > 0) {
+                        cy.wrap(body).find('#Delete').first().click();
+                        cy.wait(2000);
+                        deleteExtensions();
+                    }
+                });
+            };
+            // Cleanup
+            cy.get('#gradeable-select').select('Late Allowed Homework');
+            deleteExtensions();
+            // View bulk late day changes
+            checkStudentsInCache();
+        });
+    });
+
+    describe('Test changes to gradeable info', () => {
+        const EditGradeablePage = () => {
+            cy.visit(['sample', 'bulk_late_days']);
+            calculateCache();
+            cy.visit(['sample', 'gradeable', 'late_allowed_homework', 'update?nav_tab=5']);
+            cy.get('.breadcrumb > span').should('have.text', 'Edit Gradeable');
+        };
+
+        it('Changes gradeable due date information', () => {
+            cy.login('instructor');
+            //Changes due date
+            EditGradeablePage();
+            cy.get('#date_due')
+                .clear()
+                .type('1972-01-02 11:59:59')
+                .click();
+            cy.get('#late_days').click(); // Dismiss calender and trigger save
+            cy.get('#save_status', {timeout:10000}).should('have.text', 'All Changes Saved');
+            cy.visit(['sample', 'bulk_late_days']);
+            // Gradeable # of late days used should be empty
+            cy.get('#late-day-table > tbody > tr > [header_id="Late Allowed Homework"]')
```

Comment:
```suggestion
            cy.get('[header_id="Late Allowed Homework"]')
```
Would this selector still work ?

similar for other selectors in PR

Reply:
I tried to do assign a unique id to the header ( in this case the "Late Allowed Homework"), but it needs to first find #late-day-table

Reply:
It should find the header_id still? that's the entire point of adding another selector, as to not have to traverse the page by element, but find the specific one(s)